src/components/analyses/toolbar/AnalysesDotMenu.js

```diff
@@ -81,6 +82,7 @@ function DotMenuItems(props) {
                 id={build(baseId, ids.MENUITEM_RELAUNCH)}
                 onClick={() => {
                     onClose();
+                    handleRelaunch(selectedAnalyses[0]);
```

Comment:
Are we still supporting relaunching multiple analyses? In the old DE, it just displays a message about what relaunching multiple analyses is intended to do instead of showing the app launch wizard.

Reply:
Ah yes! Good catch. I need to add support for that and call a different endpoint, instead of routing to the relaunch page.
I'll switch this to `Draft` in the meantime.